en-us/give-up-optifine.md

```diff
@@ -1,36 +1,56 @@
+# Give up OptiFine
+
 One of the goals of Fabulously Optimized is to provide feature parity with [OptiFine](https://optifine.net/home), in order to make it easier for users to transition away from it. 
 
 Here is the list of OptiFine features that are supported in this modpack:
 
-* Performance optimizations - [Sodium](https://www.curseforge.com/minecraft/mc-mods/sodium), [Lithium](https://www.curseforge.com/minecraft/mc-mods/lithium), [Starlight](https://www.curseforge.com/minecraft/mc-mods/starlight) [etc.](https://github.com/Fabulously-Optimized/fabulously-optimized#smooth)
-* Shaders - [Iris](https://www.curseforge.com/minecraft/mc-mods/irisshaders) ([tutorial](./getting-shaders.md))
-* Capes - [Fabric Capes](https://www.curseforge.com/minecraft/mc-mods/capes) ([tutorial](./free-cape.md))
-* Zoom - [Zoomify](https://www.curseforge.com/minecraft/mc-mods/zoomify)
-* Better Grass/Snow¹ - [LambdaBetterGrass](https://www.curseforge.com/minecraft/mc-mods/irisshaders)
-* Better Trees² - [Cull Less Leaves](https://www.curseforge.com/minecraft/mc-mods/cull-less-leaves)
-* Dynamic Lights - [LambDynamicLights](https://www.curseforge.com/minecraft/mc-mods/lambdynamiclights)
-* Connected Textures - [Continuity](https://www.curseforge.com/minecraft/mc-mods/continuity)
-* Animated Textures - [Animatica](https://www.curseforge.com/minecraft/mc-mods/animatica)
-* Custom GUIs - [OptiGUI](https://www.curseforge.com/minecraft/mc-mods/optigui)
-* Custom Items - [CIT Resewn](https://www.curseforge.com/minecraft/mc-mods/cit-resewn)
-* Random Mobs, Emissive Mobs - [Entity Texture Features](https://www.curseforge.com/minecraft/mc-mods/entity-texture-features-fabric)
-* Custom Mobs/Entities - [Custom Entity Models](https://www.curseforge.com/minecraft/mc-mods/custom-entity-models-cem)
-* Custom Colors - [Colormatic](https://www.curseforge.com/minecraft/mc-mods/colormatic)
-* Toggles for animations, particles etc. - [Sodium Extra](https://www.curseforge.com/minecraft/mc-mods/sodium-extra)
-* HD Fonts - support in vanilla since 1.13, some resourcepacks that provide it [are available on my profile](https://www.curseforge.com/members/robotkoer/projects)
-* HD Screenshots - [Fabrishot](https://www.curseforge.com/minecraft/mc-mods/fabrishot)
-* Fix for transparent gaps in items - [Item Model Fix](https://www.curseforge.com/minecraft/mc-mods/item-model-fix)
-* Resource pack-customizable splash screen - [Puzzle](https://www.curseforge.com/minecraft/mc-mods/puzzle)
+## Video Options
+
+| Option | Is supported? | Mod providing the feature |
+|-|-|-|
+| Smooth Lighting Level | ❌ | There's [an  issue](https://github.com/FlashyReese/sodium-extra-fabric/issues/125) in the [Sodium Extra mod](https://www.curseforge.com/minecraft/mc-mods/sodium-extra).
```

Comment:
Please prioritize in order:
1. supported
2. partly supported
3. soon supported
4. not supported

Reply:
you added emoji but didn't change the order

Reply:
> you added emoji but didn't change the order

lol, my bad